modular_skyrat/modules/modular_vending/code/vending.dm

```diff
@@ -14,8 +16,38 @@
 	if(skyrat_contraband)
 		contraband += skyrat_contraband
 
+	/// Time to make clothes amounts consistent!
+	for (var/obj/item/clothing/item in products)
+		if(products[item] < MINIMUM_CLOTHING_STOCK && is_item_increase_banned(item))
+			products[item] = MINIMUM_CLOTHING_STOCK
+	for (var/obj/item/clothing/item in premium)
+		if(premium[item] < MINIMUM_CLOTHING_STOCK && is_item_increase_banned(item))
+			premium[item] = MINIMUM_CLOTHING_STOCK
+
 	QDEL_NULL(skyrat_products)
 	QDEL_NULL(skyrat_premium)
 	QDEL_NULL(skyrat_contraband)
 	return ..()
 
+/// This proc checks for forbidden traits cause it'd be pretty bad to have 5 insuls available to assistants roundstart at the vendor!
+/obj/machinery/vending/proc/is_item_increase_banned(obj/item/clothing/clothing_path)
```

Comment:
proc name is really long and somewhat misleading.

Reply:
if you don't change the return trues to falses and vice versa, maybe rename it to allow_increase() or something